src/multi-column-adapter/src/test/scala/MultiColumnAdapterSpec.scala

```diff
@@ -3,47 +3,65 @@
 
 package com.microsoft.ml.spark
 
-import org.apache.spark.ml.feature.Tokenizer
+import org.apache.spark.ml.feature.{StringIndexer, Tokenizer}
 import com.microsoft.ml.spark.schema.DatasetExtensions._
-import org.apache.spark.ml.Transformer
+import org.apache.spark.ml.{Estimator, PipelineModel}
 import org.apache.spark.sql.DataFrame
 import org.apache.spark.sql.types.StructType
 
-class MultiColumnAdapterSpec extends TransformerFuzzingTest {
+import scala.collection.mutable
 
+class MultiColumnAdapterSpec extends EstimatorFuzzingTest {
```

Comment:
This should extend a different fuzzer. do a git fetch upstream then git rebase upstream/master to see

Reply:
Done